_dev/apps/ui/src/store/modules/app/actions.ts

```diff
@@ -119,4 +118,12 @@ export default {
         });
     }
   },
+
+  async [ActionsTypes.GET_BANNER_SUCCESS_LOCAL_STORAGE_MONETIZATION]({commit}: Context, value) {
```

Comment:
for me it's not a GET but a SET.
And I think you can update the local storage her and not inside vue scripts. That's avoid duplication code.